src/ucs/debug/debug.c

```diff
@@ -56,11 +68,12 @@ struct backtrace_file {
     asymbol                  **syms;
 };
 
-typedef struct backtrace *backtrace_h;
 struct backtrace {
     struct backtrace_line    lines[BACKTRACE_MAX];
     int                      size;
     int                      position;
+    int                      exclude;
```

Comment:
why need 'exclude' field?

Reply:
as I can see original algorithm exclude frames on top of stack, I just reproduced this algo.
make it same as for non-detailed stack? there all frames are processed for exclusion pattern

Reply:
doesn't seem like ->exclude field is really used like that, it's just being set to 1 and that's it
you are already doing ->position = strip for this